unitTests/ui/AuthenticationProviderTest.ts

```diff
@@ -195,6 +195,7 @@ export function AuthenticationProviderTest() {
         const logggerSpy = spyOn(test.cmp.logger, 'error');
         triggerAfterComponentsInitialization();
 
+        await Promise.resolve();
         await Promise.resolve();
```

Comment:
Adding [this then chain](https://github.com/coveo/search-ui/pull/1765/files#diff-17ddd73583344050f04a376ac6631defe9aa8cc84e12ca323b1c4619488a2279R176) failed this test unless there was either a second `Promise.resolve` or a `setTimeout` 🤷

Reply:
you can use a `resolveAfter`

Reply:
oh! I'll give that a try

Reply:
oh! I'll give that a try, thanks